### PR TITLE
Return of the meta dropdown

### PR DESCRIPTION
**Pull request by jordan-quinn-alteryx**

### CODE REVIEW

File: harness/Adapter/index.tsx [hunk 1]
```diff
@@ -213,18 +212,22 @@ class Adapter extends Component<IAdapterProps, IAdapterState> {
...
             </Grid>
-
             <Divider orientation="vertical" />
             <Grid item xs className={classes.fullHeight}>
               <Paper className={classes.fullHeight}>
-              
                 <Grid container justify="center">
                   <Grid item xs="auto">
                     <Box m={4}>
+                      <TextField select noAutoWidth onChange={this.handleInputChange}defaultValue="none" disableClearable disableSearchable>
```
Comment (williammalone) [inline harness/Adapter/index.tsx:221]: Seems like a there is a missing space before `defaultValue`.

Comment (williammalone) [reply harness/Adapter/index.tsx:221]: Are `disableClearable` and `disableSearchable` valid `TextField` props? I am thinking they might be left over props from the previous `Select`.

**Review by williammalone — CHANGES_REQUESTED** [review]

File: harness/Adapter/index.tsx (hunk 1 above)
Comment (jordan-quinn-alteryx) [reply harness/Adapter/index.tsx:221]: good catch on the missing space and I thought about those two props as soon as I opened the PR as I was heading out for the evening. Updating now.

File: harness/Adapter/index.tsx [hunk 2]
```diff
@@ -73,21 +70,20 @@ const inputOptions = [{
 
 const exampleData = {
   mtg: mtg,
-  address: address
+  address: address,
```
Comment (williammalone) [inline harness/Adapter/index.tsx:73]: trailing comma, which may become a lint issue in the future if the default linter is implemented

File: harness/Adapter/index.tsx [hunk 3]
```diff
@@ -213,18 +213,24 @@ class Adapter extends Component<IAdapterProps, IAdapterState> {
...
             <Divider orientation="vertical" />
             <Grid item xs className={classes.fullHeight}>
               <Paper className={classes.fullHeight}>
-              
                 <Grid container justify="center">
-                  <Grid item xs="auto">
+                  <Grid xs={5} item>
                     <Box m={4}>
+                      <FormControl fullWidth>
+                        <TextField select label="Meta" SelectProps={{ autoWidth: false }} onChange={this.handleInputChange} defaultValue="address">
```
Comment (williammalone) [inline harness/Adapter/index.tsx:223]: Can we default to "No Input"?

File: harness/Adapter/index.tsx [hunk 4]
```diff
@@ -73,21 +70,20 @@ const inputOptions = [{
...
+  address: address,
 }
 
 class Adapter extends Component<IAdapterProps, IAdapterState> {
   constructor(props) {
     super(props);
     this.state = {
       darkMode: false,
-      model: { Configuration: {} },
+      model: { Configuration: {}, Meta: { fields: address } },
```
Comment (williammalone) [inline harness/Adapter/index.tsx:81]: Can we default to empty Meta?

**Review by williammalone — CHANGES_REQUESTED** [review]

File: harness/Adapter/index.tsx (hunk 3 above)
Comment (jordan-quinn-alteryx) [reply harness/Adapter/index.tsx:223]: We sure can!

File: harness/Adapter/index.tsx (hunk 2 above)
Comment (jordan-quinn-alteryx) [reply harness/Adapter/index.tsx:73]: Good catch

File: harness/Adapter/index.tsx (hunk 4 above)
Comment (jordan-quinn-alteryx) [reply harness/Adapter/index.tsx:81]: yup yup!

File: harness/Adapter/index.tsx [hunk 5]
```diff
@@ -62,8 +62,8 @@ const styles = (theme) => ({
 
 const inputOptions = [{
   value: 'none',
-  primary: 'No Input'
-}, {
+  primary: 'No Value'
```
Comment (williammalone) [inline harness/Adapter/index.tsx:65]: I prefer 'No Input' here since it provides no Input tool.

Comment (jordan-quinn-alteryx) [reply harness/Adapter/index.tsx:65]: That makes sense

**Review by williammalone — APPROVED** [review]

**Pull request merged**